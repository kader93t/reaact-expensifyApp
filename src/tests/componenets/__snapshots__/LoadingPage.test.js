import React from 'react';
import { shallow } from "enzyme";
import LoadingPage from "../../../components/LodingPage"

test('should render the loading page correctly', () => {
    const wrapper = shallow(<LoadingPage />);
    expect(wrapper).toMatchSnapshot();
});
