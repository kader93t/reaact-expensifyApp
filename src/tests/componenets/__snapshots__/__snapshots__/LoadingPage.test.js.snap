// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render the loading page correctly 1`] = `
<div
  className="loading-page"
>
  <img
    className="loading-page-img"
    src="/images/loader.gif"
  />
</div>
`;
